examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OverviewFragment.kt

```diff
@@ -113,6 +115,27 @@ class OverviewFragment : Fragment(), OfferingCardAdapter.OfferingCardAdapterList
         }
     }
 
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+    override fun onResume() {
```

Comment:
Could have done this on the Activity, but preferred to do it in the fragment to show how it would be to handle at the UI part of the app (whether it's using compose or fragments)